.gitignore

```diff
@@ -1,9 +1,12 @@
 /_build
-/biocaml.install
-/.merlin
-/.ocamlinit
-/*.byte
-/*.native
+/_build.prev
+/_esy
+/node_modules
+/_cache
+/_var
```

Comment:
I don't think `_var` is a directory that could get created here.

Reply:
Done: https://github.com/biocaml/biocaml/compare/b5f164db11cbb27685323c1b6c35faf4b7002b9c..f3d5feef07881fc97c4cd92793116ce63e88addd